docs/src/appendix/sncast/account/create.md

```diff
@@ -5,7 +5,7 @@ Account information will be saved to the file specified by `--accounts-file` arg
 which is `~/.starknet_accounts/starknet_open_zeppelin_accounts.json` by default.
 
 ## `--name, -n <ACCOUNT_NAME>`
-Required.
+Optional.
```

Comment:
Maybe let's extend the description below and say that if `--name` is not provided, it will be generated automatically.